src/pages/user-and-groups/create-group/create-group.spec.data.ts

```diff
@@ -1,36 +1,33 @@
-export const mockCreateGroupRes = {
+export const mockCreateorremoveGroupRes = {
 
     viewLoadEvent: {
         isUploading: false
     },
 
     sessionMock: {
-        "access_token": "mLS2rHUKCdx_FXBfYjM7MPil3U2TcXocteHwGjQPuwSAsBhM4b1FFvIwitSLl2SiaqBenITY5ll-3340Tvg",
```

Comment:
remove the access_token values from the spect data and you can put "sample_access_token" ,"sample_user_id"